src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/CoffeeMaker.java

```diff
@@ -27,6 +27,75 @@ public CoffeeMaker() {
         deliveryPipe = new DeliveryPipe();
     }
 
+    /**
+     * Fill the boiler with cups of water
+     */
+    public void fillBoilerWithWater(final int cupsWater) {
+        boiler.setCupsOfWater(cupsWater);
+    }
+
+    /**
+     * Fill the boiler with cups of water
```

Comment:
Fill the boiler with cups of water ??????????
This is not a good reference for the method

Reply:
Thanks for the observation